crates/pallet-subspace/src/lib.rs

```diff
@@ -1307,6 +1309,11 @@ fn current_vote_verification_data<T: Config>(is_block_initialized: bool) -> Vote
     let solution_ranges = SolutionRanges::<T>::get();
     VoteVerificationData {
         solution_range: if is_block_initialized {
+            solution_ranges.current
+        } else {
+            solution_ranges.next.unwrap_or(solution_ranges.current)
+        },
+        vote_solution_range: if is_block_initialized {
             solution_ranges.voting_current
         } else {
             solution_ranges
```

Comment:
@nazar-pc : `vote_solution_range` for uninitialised blocks should move to next voting solution instead of the entire solution_range. 
Implementation for `voting_solution_range` for uninitialised  blocks could be similar to `solution_range` as `solution_ranges.next.unwrap_or(solution_ranges.current)`

Reply:
The else clause for the `vote_solution_range` should be similar implementation as `solution_range`  at L1314. The vote solution should select next vote solution range or current

Reply:
They do look the same to me :confused: 

One is
```rust
solution_ranges.next.unwrap_or(solution_ranges.current)
```

The other is
```rust
solution_ranges.voting_next.unwrap_or(solution_ranges.voting_current)
```

I still don't see what is wrong here.

Reply:
Thanks for the clarification, It's good. 
The lines break below were clipped in PR , made me raise this as an issue. Apologies
<img width="720" alt="Screenshot 2023-11-13 at 14 32 28" src="https://github.com/subspace/subspace/assets/99250300/63fd02e5-639f-4d3f-b901-0446c848f6c9">